components/ingest-service/migration/migration.go

```diff
@@ -150,6 +154,17 @@ func (ms *Status) Start() error {
 		return nil
 	}
 
+	exists, err = ms.hasActionData()
+	if err != nil {
+		ms.updateErr(err.Error(), "Unable to detect migration status")
+		return err
+	}
+	if exists {
+		ms.update("Starting migration of actions to the event-feed-service")
+		go ms.migrateAction()
```

Comment:
The migration runs in the background and does not prevent the ingest-service for starting fully while migrating the actions.